bndtools.core/bnd.bnd

```diff
@@ -1,4 +1,4 @@
-# Set javac settings from JDT prefs
+	# Set javac settings from JDT prefs
```

Comment:
Revert this whitespace change

Reply:
FYI: I don't think I explicitly made these whitespace changes - I think the Bndtools editor did while I was working on it. :smile:

I'm happy to revert, but I'm curious about the reason. Is it because you don't want unrelated changes as part of this commit? Or is it because you liked the whitespace the way it was originally?

Reply:
My personal preference is that whitespace or formatting-only changes are not commingled with functional changes. It makes reviewing the functional changes easier.